addons/l10n_fr_pos_cert/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.l10n_fr_pos_cert</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="point_of_sale.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <form>
+                <field name="country_code" invisible="1"/>
+            </form>
+            <xpath expr="//div[@class='o_setting_left_pane mt-4']" position="attributes">
```

Comment:
the xpath is too generic, "o_setting_right_pane" is present hundreds of time in the settings (all settings of all modules are aggregated in one single form view with tabs) and mt-4 is just a style class. So nothing prevent custom code (or even our code) to have a node that will be matched before "Use QR code on ticket" option.

better XPath in my opinion would be eg. for the field div:
```
//field[@name='point_of_sale_use_ticket_qr_code']/..
```
for the label div:
```
//label[@for='point_of_sale_use_ticket_qr_code']/..
```